operator/main.go

```diff
@@ -274,8 +274,12 @@ func runOperator() {
 
 	// Register the CRDs after validating that we are running on a supported
 	// version of K8s.
-	if err := k8s.RegisterCRDs(); err != nil {
-		log.WithError(err).Fatal("Unable to register CRDs")
+	if !operatorOption.Config.SkipCRDCreation {
+		if err := k8s.RegisterCRDs(); err != nil {
+			log.WithError(err).Fatal("Unable to register CRDs")
+		}
+	} else {
+		log.Info("Skipping creation of CRDs")
```

Comment:
Hmm, maybe it's a good idea to move `operatorOption.Config.SkipCRDCreation` and remove the that is in there `option.Config.SkipCRDCreation`. As well as remove the SkipCRDCreation variable from the option.Config altogether.

Reply:
@aanm I wasn't able to parse your sentence, could you rephrase please?

Reply:
@kaworu I've edited the comment.

Reply:
@aanm tbh, I thought about it, but `RegisterCRDs` is also [used by](https://docs.cilium.io/en/v1.9/cmdref/cilium-agent/) cilium-agent which also provides `--skip-crd-creation`. Since I'm not an expert in the code, I decided to keep the common function untouched.
If you think it is reasonable to do so, I can implement it in another PR.

Reply:
@Fedosin The `RegisterCRDs` are not used by the cilium-agents anymore, that's why I suggested to also remove the flag option from it.

Reply:
@aanm alright, if you think it is safe, then I'll update my PR later today